proto/cometbft/types/v1/evidence.proto

```
@@ -0,0 +1,41 @@
+syntax = "proto3";
+package cometbft.types.v1;
+
+option go_package = "github.com/cometbft/cometbft/api/cometbft/types/v1";
+
+import "cometbft/types/v1/types.proto";
```

Buf (1.47.2)

import "cometbft/types/v1/types.proto": file does not exist

(COMPILE)